conda_smithy/configure_feedstock.py

```diff
@@ -100,17 +102,24 @@ def render_circle(jinja_env, forge_config, forge_dir):
 
 @contextmanager
 def fudge_subdir(subdir):
```

Comment:
Ping @msarahan on this just so you know. I have added a few tests for the function which should help with tracking this kind of issue down in the future.